content/docs/user-guide/data.md

```diff
@@ -1,36 +1,18 @@
-# Working with data
+# Working with Data
 
-<details>
+You need to save data as a [MLEM Object] so that it is compatible with
+operations such as [mlem apply](/doc/command-reference/apply).
```

Comment:
So far I simplified and focused this guide (guides don't need to be reproducible) based on the existing content.

But really there's a bigger "problem" with it: It seems like just another example of https://mlem.ai/doc/get-started/saving. We probably don't need that redundancy. The guide should focus more on how things work so for example, the schema of `.mlem` files, limitations, alternate scenarios (different `save()` arg combinations).

For example see how https://dvc.org/doc/start/experiments differs from https://dvc.org/doc/user-guide/experiment-management/running-experiments.